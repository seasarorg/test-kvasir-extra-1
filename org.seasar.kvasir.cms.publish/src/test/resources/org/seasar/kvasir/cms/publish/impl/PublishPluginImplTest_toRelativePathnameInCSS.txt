.test {
    a: url(/a/c/d.html);
    b: url('/a/c/d.html');
    c: url("/a/c/d.html");
}
