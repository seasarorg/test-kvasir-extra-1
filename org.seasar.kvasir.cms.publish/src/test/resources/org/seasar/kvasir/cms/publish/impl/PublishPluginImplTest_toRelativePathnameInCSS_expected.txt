.test {
    a: url(../c/d.html);
    b: url('../c/d.html');
    c: url("../c/d.html");
}
